src/gui/editors/PianoRoll.cpp

```diff
@@ -788,9 +775,19 @@ QColor PianoRoll::markedSemitoneColor() const
 void PianoRoll::setMarkedSemitoneColor( const QColor & c )
 { m_markedSemitoneColor = c; }
 
-void PianoRoll::drawNoteRect(QPainter & p, int x, int y, 
+float PianoRoll::noteOpacity() const
+{ return m_noteOpacity; }
+
+void PianoRoll::setNoteOpacity( const float & f )
```

Comment:
I feel like this makes it harder to tell how high a notes velocity is. For
example, two identical notes on top of each other vs one note with twice
the velocity.

Could we maybe use note color to denote velocity?